.github/workflows/functional-networking.yaml

```diff
@@ -0,0 +1,73 @@
+name: functional-networking
+on:
+  pull_request:
+    paths:
+      - '^.*networking.*$'
+      - '.github/workflows/functional-networking.yaml'
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  functional-networking:
+    strategy:
+      fail-fast: false
+      matrix:
+        name: ["master"]
+        openstack_version: ["master"]
+        ubuntu_version: ["20.04"]
+        devstack_conf_overrides: [""]
+        include:
+          - name: "xena"
+            openstack_version: "stable/xena"
+            ubuntu_version: "20.04"
+            devstack_conf_overrides: ""
+          - name: "wallaby"
+            openstack_version: "stable/wallaby"
+            ubuntu_version: "20.04"
+            devstack_conf_overrides: ""
+          - name: "victoria"
+            openstack_version: "stable/victoria"
+            ubuntu_version: "20.04"
+            devstack_conf_overrides: ""
+          - name: "ussuri"
+            openstack_version: "stable/ussuri"
+            ubuntu_version: "18.04"
+            devstack_conf_overrides: |
+              enable_plugin neutron-fwaas https://git.openstack.org/openstack/neutron-fwaas stable/ussuri
+          - name: "train"
+            openstack_version: "stable/train"
+            ubuntu_version: "18.04"
+            devstack_conf_overrides: |
+              enable_plugin neutron-fwaas https://git.openstack.org/openstack/neutron-fwaas stable/train
+    runs-on: ubuntu-${{ matrix.ubuntu_version }}
+    name: Deploy OpenStack ${{ matrix.name }} with Neutron and run networking acceptance tests
+    steps:
+      - name: Checkout Gophercloud
+        uses: actions/checkout@v2
+      - name: Deploy devstack
+        uses: EmilienM/devstack-action@v0.6
+        with:
+          branch: ${{ matrix.openstack_version }}
+          conf_overrides: |
+            Q_ML2_PLUGIN_EXT_DRIVERS=qos,port_security,dns_domain_keywords
+            enable_plugin neutron-vpnaas https://opendev.org/openstack/neutron-vpnaas ${{ matrix.openstack_version }}
+            ${{ matrix.devstack_conf_overrides }}
+          enabled_services: 'neutron-dns,neutron-qos,neutron-segments,neutron-trunk,neutron-uplink-status-propagation,neutron-network-segment-range,neutron-port-forwarding'
```

Comment:
It seems like the `port-forwarding` extension is not enabled: https://github.com/gophercloud/gophercloud/runs/4928771066?check_suite_focus=true#step:5:3055
Same goes for the `vlan-transparent` extension: https://github.com/gophercloud/gophercloud/runs/4928771066?check_suite_focus=true#step:5:4069

On train, in addition to the above extensions, we're also missing:
- dns-integration: https://github.com/gophercloud/gophercloud/runs/4928771304?check_suite_focus=true#step:5:2589
- qos: https://github.com/gophercloud/gophercloud/runs/4928771304?check_suite_focus=true#step:5:3353
- trunk: https://github.com/gophercloud/gophercloud/runs/4928771304?check_suite_focus=true#step:5:3574

Reply:
Honestly, I'm not sure how to enable VLAN transparent, and I'm not even sure it works for OVN.
As for Train, I'm sure it's because of the default Neutron plugins (ML2/OVS). We could change that in the future if you don't mind, but I think the coverage is pretty good now.